app/controllers/third_party_employment.py

```diff
@@ -129,4 +129,10 @@ def resolve_employment_token(self, info, _, client_id, employment_id):
             ThirdPartyClientEmployment.client_id == client_id,
             ~ThirdPartyClientEmployment.is_dismissed,
         ).one_or_none()
+
+        if not check_protected_client_id_company_id(
+            client_employment_link.employment.company_id
+        ):
```

Comment:
:orange_circle: Cela risque de renvoyer une erreur si le client_employment_link n'existe pas : 
```suggestion
        if client_employment_link and not check_protected_client_id_company_id(
            client_employment_link.employment.company_id
        ):
```